packages/blockchain-link-types/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0
+
+-   feat(blockchain-link): cardano name and ticker (8e15fa0505)
```

Comment:
weren't we supposed to release from the same commit we have released the previous canary from? this way it beats the purpose of canary a little bit